iOS/Old UI/Manga/Tracking/TrackerListView.swift

```diff
@@ -17,7 +17,7 @@ struct TrackerListView: View {
     var body: some View {
         VStack {
             ForEach(TrackerManager.shared.trackers, id: \.id) { tracker in
-                if tracker.isLoggedIn {
+                if tracker.isLoggedIn && (tracker.restrictSources == nil || tracker.restrictSources!.contains(manga.sourceId) ){
```

Comment:
avoid `!` if you can
```suggestion
                if tracker.isLoggedIn && (tracker.restrictSources ?? []).contains(manga.sourceId) {
```